core/src/main/java/oracle/weblogic/deploy/util/WLSDeployArchive.java

```diff
@@ -58,6 +58,12 @@ public class WLSDeployArchive {
      */
     public static final String ARCHIVE_APPS_TARGET_DIR = WLSDPLY_ARCHIVE_BINARY_DIR + "/applications";
 
+    /**
+     * Top-level archive subdirectory where the applications are stored and the subdirectory to which
+     * they will be extracted. This is for structured applications found under /app
+     */
+    public static final String ARCHIVE_APPS_FOLD_TARGET_DIR = WLSDPLY_ARCHIVE_BINARY_DIR + "/application_folder";
```

Comment:
Typically, we use plural nouns for the folders in the archive, please change to make this consistent.